recipe/install_nvcc.sh

```diff
@@ -91,18 +91,31 @@ export CMAKE_ARGS="\${CMAKE_ARGS}"
 ### /CMake configurations
 
 mkdir -p "\${CONDA_BUILD_SYSROOT}/lib"
+mkdir -p "\${CONDA_PREFIX}/lib/stubs"
 
 # Add \$(libcuda.so) shared object stub to the compiler sysroot.
 # Needed for things that want to link to \$(libcuda.so).
 # Stub is used to avoid getting driver code linked into binaries.
 
-# Make a backup of \$(libcuda.so) if it exists
-if [[ -f "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" ]]
+
+if [[ "\${CONDA_BUILD}" == 1 ]]
 then
-  LIBCUDA_SO_CONDA_NVCC_BACKUP="\${CONDA_BUILD_SYSROOT}/lib/libcuda.so-conda-nvcc-backup"
-  mv "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" "\${LIBCUDA_SO_CONDA_NVCC_BACKUP}"
+  # Make a backup of \$(libcuda.so) if it exists
+  if [[ -f "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" ]]
+  then
+    LIBCUDA_SO_CONDA_NVCC_BACKUP="\${CONDA_BUILD_SYSROOT}/lib/libcuda.so-conda-nvcc-backup"
+    mv "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" "\${LIBCUDA_SO_CONDA_NVCC_BACKUP}"
+  fi
```

Comment:
Does this mean that even if a user has libcuda.so installed in their conda environment we will always ignore it and force a fallback to the stub pointing to the host libcuda.so? Based on @robertmaynard's reply [here](https://github.com/conda-forge/nvcc-feedstock/pull/84#issuecomment-1138672547) I thought we didn't want that?

Reply:
Just saw the conditional. So we're forcing use of the system libcuda.so in conda build environments, but not in user space. If a user installs libcuda.so into their own conda environment (not during conda build) then we do use that, just not during conda builds. Is that right?

Reply:
> If a user installs libcuda.so into their own conda environment (not during conda build) then we do use that, just not during conda builds. Is that right?

From what @kkraus14 mentioned above, if they have the `.so` in this location already then they are on their own. We force symlink currently to the host (or) whatever the `CUDA_HOME` points to.